docs/operations/api-reference.md

```diff
@@ -614,6 +614,11 @@ Retrieve information about the segments of a task.
 
 Retrieve a [task completion report](../ingestion/tasks.md#task-reports) for a task. Only works for completed tasks.
 
+* `/druid/indexer/v1/lockedIntervals`
```

Comment:
Is this supposed to be called by users? What is the use case?

Reply:
Actually, instead of exposing this API, maybe a better thing we need is how to communicate to the user that compaction skips an intervals because of some task holding lock. Like what intervals was skipped for which datasource and what was the task that holds the lock.

Reply:
For now, we are just logging the intervals that were skipped due to locks.
The part about notifying the user about skipped intervals will most likely be done through Task Reports in a follow up PR.

Reply:
> Is this supposed to be called by users? What is the use case?
Removed changes to `api-reference.md` as this API is for internal use (between Coordinator and Overlord) only.